src/Controls/src/Core/Handlers/Items/iOS/CarouselViewController.cs

```diff
@@ -134,6 +134,9 @@ public override void UpdateItemsSource()
 
 		protected override UICollectionViewDelegateFlowLayout CreateDelegator() => new CarouselViewDelegator(ItemsViewLayout, this);
 
+#if NET8_0_OR_GREATER
+		[Obsolete("Use DetermineCellReuseId(NSIndexPath indexPath) instead.")]
+#endif
 		protected override string DetermineCellReuseId()
```

Comment:
Does it matter that this code is not running anymore? Or maybe I missed something?

Reply:
It's using the alternate version of the method, so no; it doesn't matter.